draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1914,8 +1913,9 @@ outlined in [RFC5226].
 ## New Entry to the TLS ExtensionType Registry
 
 IANA is asked to add an entry for `transparency_info(TBD)` to the "TLS
-ExtensionType Values" registry defined in [I-D.ietf-tls-tls13], citing this
-document as the "Reference" and setting the "Recommended" value to "Yes".
+ExtensionType Values" registry defined in [RFC8446], setting the "Recommended"
+value to "Yes", setting the "TLS 1.3" value to "CH, CR, CT", and citing this
```

Comment:
We're using single letters now, "Y".  Don't blame me, blame DTLS-OK...

Reply:
Thanks @kaduk.  Addressed by https://github.com/google/certificate-transparency-rfcs/pull/298/commits/b2f37ae57279c70eef9cb917ba5ae38e345d9aa6